pkg/apis/tensorflow/v1/defaults_test.go

```diff
@@ -55,10 +55,12 @@ func expectedTFJob(cleanPodPolicy common.CleanPodPolicy, restartPolicy common.Re
 
 	return &TFJob{
 		Spec: TFJobSpec{
-			SuccessPolicy:  &defaultSuccessPolicy,
-			CleanPodPolicy: &cleanPodPolicy,
-			TFReplicaSpecs: map[TFReplicaType]*common.ReplicaSpec{
-				TFReplicaTypeWorker: &common.ReplicaSpec{
+			SuccessPolicy: &defaultSuccessPolicy,
```

Comment:
I am not sure if success policy is part of run policy. This is something we can consolidate together in future releases

Reply:
Currently, SuccessPolicy and FailurePolicy(we have opened an issue for it) are for distributed tensorflow only, I am not sure the definition of Success and Failure are the same for other framework.